### ISSUE TITLE
Ignore template rendering directory

### ISSUE THREAD

**Issue by vDMG** [gruntwork-io/boilerplate#167]
First, thanks for open-sourcing this super cool project ! 

**Describe the solution you'd like**
We need a method to disable the template rendering for certain files/folders, especially when dealing with components like Helm chart templates that already include Go templates. In such cases, we don't want the 'boilerplate' to render them; instead, we prefer it to output them as-is.

**Describe alternatives you've considered**
Not sure this is a clean way to do this.

**Additional context**
I propose to add a property like `disable_templating` or `skip_templating` to the boilerplate manifest working the same way `skip_files` works.

WDYT ? If you like this proposal i'd love to contribute to the project.

**Comment by brikis98** [comment]
Did you try [`skip_files`](https://github.com/gruntwork-io/boilerplate?tab=readme-ov-file#skip-files)?

**Comment by vDMG** [comment]
> Did you try [`skip_files`](https://github.com/gruntwork-io/boilerplate?tab=readme-ov-file#skip-files)?

Yes, but `skip_files` ignore those files and they does not appear on the rendered project. In my case, I want to disable templating but I need them in the rendered project. So just a simple copy/paster without trying to template them.

**Comment by brikis98** [comment]
Oh, I gotcha. Sorry, read to quickly, and misunderstood. 

A `skip_templating` feature similar to `skip_files` makes sense to me! PR welcome.

**Comment by comminutus** [comment]
Any chance this can be pushed through? I'd like to use this project for some projects but I have some Ansible scripts that need to be included in the templated project that need to be skipped from templating, similar to @vDMG 's problem.  I don't quite understand the workaround, since it doesn't seem like it would avoid boilerplating from trying to template the files inside of the project directory.
